### PR TITLE
Reduce server card size when there are multiple servers

### PR DESCRIPTION
**Pull request by alalamav**
* Condenses the server card UI when there are more than one server.
  * Implements a responsive server list, with a threshold of 360px screen width.
* The UI remains unchanged if there are no servers (intro screen) or one server (full size single server card).
* [Demo video](https://drive.google.com/file/d/1cWrxUGm1m9zWQZQHq9-VbpMl-ZDivnzM/view?usp=sharing).

### CODE REVIEW

File: src/www/ui_components/server-connection-viz.html [hunk 1]
```diff
@@ -167,26 +188,49 @@
...
+          width: 8px;
+        }
+        #md,
+        #md-grey {
+          top: 8px;
+          left: 8px;
+          height: 16px;
+          width: 16px;
+        }
+        #lg,
```
Comment (JonathanDCohen) [inline src/www/ui_components/server-connection-viz.html:206]: Nit: could we use "small", "medium", and "large" instead?

File: src/www/ui_components/server-list.html [hunk 2]
```diff
@@ -66,6 +67,10 @@
         localize: Function,
         rootPath: String,
         servers: Array,
+        hasMultipleServers: {
```
Comment (JonathanDCohen) [inline src/www/ui_components/server-list.html:70]: Nit: it's a little more readable to make this hasOneServer to avoid the extra negative.

Comment (alalamav) [reply src/www/ui_components/server-list.html]: Done.

File: src/www/ui_components/server-connection-viz.html (hunk 1 above)
Comment (alalamav) [reply src/www/ui_components/server-connection-viz.html]: Done.

**Pull request merged**